src/revivalpmmp/pureentities/PureEntities.php

```diff
@@ -161,33 +169,33 @@ public function onLoad(){
         PureEntities::$instance = $this;
     }
 
-    public function onEnable(){
+    public function onEnable() {
         $this->getServer()->getPluginManager()->registerEvents(new EventListener($this), $this);
         $this->saveDefaultConfig();
         $this->reloadConfig();
         $this->getServer()->getScheduler()->scheduleRepeatingTask(new AutoDespawnTask($this), $this->getConfig()->getNested("despawn-task.trigger-ticks", 1000));
         $this->getServer()->getScheduler()->scheduleRepeatingTask(new AutoSpawnTask($this), $this->getConfig()->getNested("spawn-task.trigger-ticks", 1000));
-	    $this->getServer()->getLogger()->notice("Enabled!");
-	    $this->getServer()->getLogger()->notice("You're Running ".$this->getDescription()->getFullName());
+        $this->getServer()->getLogger()->notice("Enabled!");
+        $this->getServer()->getLogger()->notice("You're Running " . $this->getDescription()->getFullName());
     }
 
-    public function onDisable(){
+    public function onDisable() {
         $this->getServer()->getLogger()->notice("Disabled!");
     }
 
     /**
      * @param int|string $type
      * @param Position $source
      * @param $args
-     * 
+     *
      * @return Entity
      */
-    public static function create($type, Position $source, ...$args){
+    public static function create($type, Position $source, ...$args) {
         $chunk = $source->getLevel()->getChunk($source->x >> 4, $source->z >> 4, true);
-        if(!$chunk->isGenerated()){
+        if (!$chunk->isGenerated()) {
```

Comment:
You shouldn't go around changing the formatting for everything. I have no problems with it, but it it better to have it be consistent with everything instead of only a few files